pkg/config/loader.go

```diff
@@ -495,6 +500,59 @@ func (l *Loader) handleEnableOnlyOption() error {
 	return nil
 }
 
+func (l *Loader) handleFormatters() {
+	l.handleFormatterOverrides()
+	l.handleFormatterExclusions()
+}
+
+// Overrides linter settings with formatter settings if the formatter is enabled.
+func (l *Loader) handleFormatterOverrides() {
+	if slices.Contains(l.cfg.Formatters.Enable, "gofmt") {
+		l.cfg.LintersSettings.GoFmt = l.cfg.Formatters.Settings.GoFmt
+	}
+
+	if slices.Contains(l.cfg.Formatters.Enable, "gofumpt") {
+		l.cfg.LintersSettings.GoFumpt = l.cfg.Formatters.Settings.GoFumpt
+	}
+
+	if slices.Contains(l.cfg.Formatters.Enable, "goimports") {
+		l.cfg.LintersSettings.GoImports = l.cfg.Formatters.Settings.GoImports
+	}
+
+	if slices.Contains(l.cfg.Formatters.Enable, "gci") {
+		l.cfg.LintersSettings.Gci = l.cfg.Formatters.Settings.Gci
+	}
+}
+
+// Add formatter exclusions to linters exclusions.
+func (l *Loader) handleFormatterExclusions() {
+	if len(l.cfg.Formatters.Enable) == 0 {
+		return
+	}
+
+	for _, path := range l.cfg.Formatters.Exclusions.Paths {
+		l.cfg.Linters.LinterExclusions.Rules = append(l.cfg.Linters.LinterExclusions.Rules, ExcludeRule{
+			BaseRule: BaseRule{
+				Linters: l.cfg.Formatters.Enable,
+				Path:    path,
+			},
+		})
+	}
+}
+func (l *Loader) handleFormatterDeprecations() {
```

Comment:
The `formatters` configuration is completely new. Do we really need to create deprecations for it? I know you are mimicking `linters`, and this will be removed in v2, but still, this complicates the logic.

Reply:
I did it voluntarily to ease the cleaning of the v2 code.